netkan/netkan/spacedock_adder.py

```diff
@@ -71,11 +73,28 @@ def try_add(self) -> bool:
                 self.COMMIT_TEMPLATE.safe_substitute(
                     defaultdict(lambda: '', self.info)),
                 author=git.Actor(self.info.get('username'), self.info.get('email')))
+            try:
+                self.nk_repo.pull_remote_branch(branch_name)
+            except git.GitCommandError:
+                pass
+            netkan_path.write_text(self.yaml_dump(netkan))
+            self.nk_repo.git_repo.index.add([netkan_path.as_posix()])
+            # Commit - Use Submitter username/email
+            self.nk_repo.git_repo.index.commit(
+                (
+                    f"Add {self.info.get('name')} from {self.info.get('site_name')}"
+                    f"\n\nThis is an automated commit on behalf of {self.info.get('username')}"
+                ),
+                author=git.Actor(self.info.get('username'),
+                                 self.info.get('email'))
+            )
+            self.nk_repo.push_remote_branch(branch_name)
```

Comment:
```suggestion
```

Bad merge, this block shouldn't be here (deleted in #291, we use `with self.nk_repo.change_branch` now).